app/src/main/java/de/westnordost/streetcomplete/data/osm/SimpleOverpassQuestType.java

```diff
@@ -1,5 +1,7 @@
 package de.westnordost.streetcomplete.data.osm;
 
+import android.util.Log;
```

Comment:
Fine to leave the logging so long as the debug level (Log.d?) is disabled in production!